gcam-usa.md

```diff
@@ -216,6 +216,32 @@ Non-CO2 GHGs
 ============
 Note that at this time the specification of Non-CO2 greenhouse gases in GCAM-USA is incomplete.  This is an active area of development.  However for this reason the climate model will be disabled in GCAM-USA scenarios.
 
+Air Pollutants
+============
+#### Methodology and Key Data Sources 
+GCAM-USA includes state-level emissions from the following air pollutants: BC, OC, PM2.5, PM10, NOx, SO2, NMVOC, CO, and NH3 for the following aggregate sectors: electric generation, buildings, onroad transportation, nonroad transportation, industrial energy use, industry processes, urban processes, cement, and refining and related. Depending on the emissions and energy data available, either emission factors (EFs) or input emissions are used. These EFs or emissions are calibrated in all historical years to the US [National Emissions Inventory](gcam-usa.html#nei) (NEI), allowing for state-level representations of pollutant emissions. Annual data at the state level from [EPA Tier 1 CAPS](gcam-usa.html#epatier1) was used to scale some of the NEI data to get a more accurate distribution of emissions for years in between, or before, NEI years. National totals for historical years are scaled to [Community Emissions Data System](gcam-usa.html#ceds) (CEDS) values for most sectors at the national level both for consistency over time and with the global GCAM model, and because CEDS values are updated annually, providing up to date estimates. Future emission factors are set, by vintage where available in GCAM, according to current air pollution control regulations. Outlier emission factors are reset to national median values to avoid spurious results. In some sectors, supplemental data from [MOVES](gcam-usa.html#moves), [MARKAL](gcam-usa.html#markal), [GREET](gcam-usa.html#greet), [IMO](gcam-usa.html#imo), and various other EPA sources.
+
+#### Additional Information and Assumptions
+BC and OC: The NEI does not include BC/OC emissions. These are derived from PM2.5 emissions using BC/OC ratios. There can be some inconsistency between GCAM USA BC/OC emissions and PM2.5 emissions. In practice this is usually not a significant issue because most analysis uses either BC/OC (climate-focused analysis) or PM2.5 (air pollution focused analysis) emissions, but not both.

```

Comment:
- "These are derived":  I suggest being more specific, e.g., "BC/OC emission factors are derived..."
- "There can be some inconsistency":  I suggest "This can lead to some inconsistency..."
-